test/ml/test_ml.py

```diff
@@ -42,6 +42,9 @@ def test_summarize_networks_wrong_direction(self):
         with pytest.raises(ValueError):
             ml.summarize_networks([INPUT_DIR + 'test-data-wrong-direction/wrong-direction.txt'])
 
+    def test_repeat_edges(self):
```

Comment:
Look at `run tests` in checks. The error is the expected behavior I was expecting to see.